engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -2108,6 +2256,16 @@ private void advanceStop(final VMInstanceVO vm, final boolean cleanUpEvenIfUnabl
         }
     }
 
+    private void changeVrCountResourceBy(VirtualMachine.Type type, long domainId, ServiceOffering offering, Account owner, boolean decrement) {
```

Comment:
```suggestion
    private void updateVrCountResourceBy(VirtualMachine.Type type, long domainId, ServiceOffering offering, Account owner, boolean decrement) {
```
Update is more clear IMHO.

Reply:
done.